sdk/appcenter-storage/src/main/java/com/microsoft/appcenter/storage/models/TokenResult.java

```diff
@@ -50,6 +50,13 @@
     @SerializedName(value = "status")
     private String status;
 
+    /**
+     * The utc timestamp for a token becoming invalid.
+     */
+    @Expose
+    @SerializedName(value = "ttl")
+    private long ttl;
```

Comment:
prefer to add a new property in the api response to indicate how long the token will expire

Reply:
We should probably sum up the value of `_ts` with the number of seconds we pass to the create permissions as a parameter in Token Exchange.
https://docs.microsoft.com/en-us/rest/api/cosmos-db/create-a-permission

Reply:
yup! that is what intended to do. The ttl defined here is DateTime.now() + _ts.getSeconds()